clusterloader2/testing/list/modules/measurements.yaml

```diff
@@ -3,6 +3,8 @@
 ## Input params
 # Valid actions: "start", "gather"
 {{$action := .action}}
+{{$ENABLE_CONTAINER_MEMORY_USAGE_MEASUREMENT := DefaultParam .CL2_ENABLE_CONTAINER_MEMORY_USAGE_MEASUREMENT true}}
```

Comment:
As for name, I don't think this is about memory usage measurement, it's still measured. What we are adding is a failure threshold for apiserver.

Maybe `ENABLE_APISERVER_MEMORY_USAGE_THRESHOLD`?